package.json

```diff
@@ -118,6 +118,7 @@
     "p-limit": "^2.2.0",
     "pngjs": "^3.4.0",
     "request": "^2.87.0",
+    "request-promise-native": "^1.0.8",
```

Comment:
this should be a devDependency, shouldn't it? We only use it for the snapshot generation during the SDK build itself.